manifests/filebeat_oss.pp

```diff
@@ -58,18 +58,14 @@
     require => Package['filebeat'],
   }
 
-  # TODO: use archive from puppet-archive module for this task
-  file { "/tmp/${$wazuh_filebeat_module}":
-    owner  => 'root',
-    group  => 'root',
-    mode   => '0440',
-    source => "https://packages.wazuh.com/4.x/filebeat/${$wazuh_filebeat_module}",
-  }
-  ~> exec { "Unpackaging /tmp/${$wazuh_filebeat_module}":
-    command     => "/bin/tar -xzvf /tmp/${$wazuh_filebeat_module} -C /usr/share/filebeat/module",
-    notify      => Service['filebeat'],
-    require     => Package['filebeat'],
-    refreshonly => true,
+  archive { "/tmp/${$wazuh_filebeat_module}":
+    ensure       => present,
+    source       => "https://packages.wazuh.com/4.x/filebeat/${$wazuh_filebeat_module}",
+    extract      => true,
+    extract_path => '/usr/share/filebeat/module',
+    creates      => '/usr/share/filebeat/module/wazuh',
```

Comment:
We should also set the cleanup flag to true so the downloaded file will be removed and only be downloaded again when we need it.
```suggestion
    creates      => '/usr/share/filebeat/module/wazuh',
    cleanup      => true,
```

Reply:
Done, it's more explicit like that